guides/common/modules/proc_importing-ansible-roles.adoc

```diff
@@ -2,11 +2,12 @@
 
 = Importing Ansible Roles and Variables
 
-You can import Ansible roles and variables from the `/etc/ansible/roles` directory on {Project} or on a {SmartProxy} that has Ansible enabled.
-Ensure that the roles and variables that you import are located in the `/etc/ansible/roles` directory on all {SmartProxies} from where you want to use the roles.
+You can import Ansible roles and variables from the xref:Ansible-paths_{context}[] on {ProjectServer} or {SmartProxy} that has Ansible enabled.
```

Comment:
I think if you say "ProjectServer", then "SmartProxyServer" is the better alternative here as well.